packages/babel-parser/test/fixtures/comments/basic/class-static-block/input.js

```diff
@@ -0,0 +1 @@
+(class /* 1 */ { /* 2 */ static /* 3 */ {/* 4 */} /* 5 */ } /* 6 */);
```

Comment:
Currently Babel does not attach comments `3`, `4` and `5`. It is now fixed in this PR.